packages/plugins/src/utils.ts

```diff
@@ -1,10 +1,19 @@
 import {
   ChugSplashRuntimeEnvironment,
-  readUnvalidatedChugSplashConfig,
+  ParsedContractConfig,
 } from '@chugsplash/core'
 import { HardhatRuntimeEnvironment } from 'hardhat/types'
 
-import { importOpenZeppelinStorageLayouts } from './hardhat/artifacts'
+import { importOpenZeppelinStorageLayout } from './hardhat/artifacts'
+
+const fetchOpenZeppelinStorageLayout = async (
+  hre: HardhatRuntimeEnvironment | undefined = undefined,
```

Comment:
If you agree with my comments above, we can remove this function and just use `importOpenZeppelinStorageLayout`